ui/ui.go

```diff
@@ -112,13 +112,19 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			return m, cmd
 		}
 
-		if m.prSidebar.GetIsCommenting() {
+		if m.prSidebar.GetIsCommenting() ||
+			m.prSidebar.GetIsAssigning() ||
+			m.prSidebar.GetIsUnassigning() {
```

Comment:
Maybe we can have the `prSidebar` just expose a single method called `m.prSidebar.isTextInputFocused()` or something

Reply:
Yeah, this sort of thing was one of the refactors I was thinking of exploring.